src/microbe_stage/OrganelleTemplate.cs

```diff
@@ -47,6 +47,26 @@ public OrganelleTemplate(OrganelleDefinition definition, Hex location, int rotat
     [JsonIgnore]
     public IEnumerable<Hex> RotatedHexes => Definition.GetRotatedHexes(Orientation);
 
+    public override int GetHashCode()
+    {
+        return Definition.GetHashCode() * Position.GetHashCode();
```

Comment:
Shift and XOR is commonly used for combining partial hash codes, see for example other hash code calculations in the codebase.